### PR TITLE
Add uri as an acceptable use of ODO.id

### PR DESCRIPTION
**Pull request by colbyharrison**
Addresses #89

### CODE REVIEW

**Comment by csuwildcat** [comment]
This doesn't feel necessary given the spec text already allows for URIs. We wrote the spec text to generally allow for all ID formulations because we didn't want to list all the types of strings one might imagine, just the requirements that they each be unique from each other in the scope of the document.

**Review by brentzundel — APPROVED** [review]

**Comment by brentzundel** [comment]
I agree that technically the spec can do this (a URI is def a string), but think that there is value in having a single sentence pointing it out, just to make it clear that some implementers are doing that deliberately.

**Comment by brentzundel** [comment]
@csuwildcat, what do you think?

**Comment by csuwildcat** [comment]
I'm not sure it needs to be explicitly said, because a URI already fits the definition. If we add language, perhaps it's better to add something like: "(e.g. a unique URI)"?

File: spec/spec.md [hunk 1]
```diff
@@ -151,7 +151,7 @@ _Credential Manifests_ are JSON objects composed as follows:
 
 [[ref:Output Descriptor Objects]] are composed as follows:
 
-- The [[ref:Output Descriptor Object]] ****MUST**** contain an `id` property. The value of the `id` property ****MUST**** be a string that does not conflict with the `id` of another [[ref:Output Descriptor Object]] in the same [[ref:Credential Manifest]].
+- The [[ref:Output Descriptor Object]] ****MUST**** contain an `id` property. The value of the `id` property ****MUST**** be a string that does not conflict with the `id` of another [[ref:Output Descriptor Object]] in the same [[ref:Credential Manifest]]. It is acceptable for `id` to be a URI.
```
Comment (brentzundel) [inline spec/spec.md:154]:
```suggestion
- The [[ref:Output Descriptor Object]] ****MUST**** contain an `id` property. The value of the `id` property ****MUST**** be a string that does not conflict with the `id` of another [[ref:Output Descriptor Object]] in the same [[ref:Credential Manifest]], e.g., a UUID or a unique URI.
```

**Review by csuwildcat — APPROVED** [review]

**Review by brentzundel — APPROVED** [review]

**Pull request merged**